### DISCUSSION TITLE
myFilter.cpp needs work - quotes instead of <> and a commented line defining its origin

### DISCUSSION THREAD

**Discussion by daveclifford** [DCC-EX/EX-T3-WiFi#60] [General]
Alex,

I'm building a DCC-EX Command Station and copied your EX-T3-WIFI  **myFilter.cpp** to the DCC-EX software directory (as per your **code.md** instructions).

**1. Arduino IDE V2.0.3 did not compile **myFilter.cpp** as defined Nov 19, 2022.  I had to change all include <...> lines  to include "..." lines (with quotes) so the IDE would compile.**

2. Please consider adding a commented line in the file defining the source attribution, so a user can determine where/why they included the file.

Dave

**Comment by alex-code** [comment]
Thanks for reporting, I'd moved away from Arduino IDE so hadn't tested on that.

**Reply by daveclifford** [reply]
I am building an Arduino DCC-EX Command Station to use with the EX-T3-WIFI throttles (building 2), so I added the myFilter.cpp file to the mix and discovered the compile issue.  I hope I helped others get some help and advice.

Dave

**Comment by daveclifford** [comment]
Alex

Thanks for updating myFilter.cpp. Other users wlii no longer scratch their heads and wonder how to fix the compile error.

Dave
